live-examples/js-examples/string/string-raw.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">const name = 'Bob';
+
+// Create a variable that uses a Windows
+// path without escaping the backslashes:
+const filePath = String.raw`C:\Development\profile\aboutme.html`;
+console.log('The file was uploaded from: ' + filePath);
+// expected output: "The file was uploaded from: C:\Development\profile\aboutme.html"
+</code>
+</pre>
+    
```

Comment:
Files should end with a new line, and please remove the spaces from this line.

Reply:
Fixed. I think VS Code is adding the spaces because it expects the line to be indented. I will watch out for that in the future.